chart-modules/common/picasso/chart-builder/presets/dimension-measure-chart/dimension-measure-chart.js

```diff
@@ -143,6 +143,7 @@ function addMeasureScale(chartBuilder, options, chartOrientation, measureAxisPro
 function addMeasureAxisComponent(chartBuilder, options, measureDirection, measureAxisProperties) {
   let measureAxisSettings = {
     scale: 'measure',
+    settings: options.settings || {},
```

Comment:
I would like a more specific name. (not just settings)
(maybe something like axisTitleSettings or axisTitleStyle)

Reply:
Done.